stages/org.osbuild.ostree.config

```diff
@@ -63,6 +68,9 @@ def main(tree, options):
     if readonly is not None:  # can be False, which we would want to set
         ro = "true" if readonly else "false"
         ostree.cli("config", "set", "sysroot.readonly", ro, repo=repo)
+    
+    if bls_append_except_default:
+        ostree.cli("config", "set", "sysroot.bls-append-except-default", 'grub_users=""', repo=repo)
```

Comment:
the `grub_users=""` should be something that is set by the config. i.e. bls_append_except_default should accept a string and then the input json manifest would specify what value the configuration key should be set to.